src/main/java/org/opentripplanner/routing/DefaultRoutingService.java

```diff
@@ -0,0 +1,43 @@
+package org.opentripplanner.routing;
```

Comment:
I think you can put this in `org.opentripplanner.routing.service`. I know there is a `services` package there already, not sure what to do with it - but that is another PR...
 
https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/CODE_CONVENTIONS.md#packages

Reply:
I've moved it to `service` (singular).

The stuff in `services` should probably be in dagger modules. I've started to do that in the upcoming PR.